cmd/kola/kola.go

```diff
@@ -47,18 +46,19 @@ var (
 )
 
 func init() {
-	cmdRun.Flags().StringVar(&kolaPlatform,
-		"platform", "qemu", "VM platform: qemu or gce")
-	cmdRun.Flags().StringVar(&EtcdRollingVersion,
+	sv := cmdRun.Flags().StringVar
+
+	sv(&EtcdRollingVersion,
```

Comment:
I have plans to rip these options out today once this PR goes in anyway since its all a hack for that one test.